src/components/z-tag/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Component, Prop, h, Host } from "@stencil/core";
+import classNames from "classnames";
+
+@Component({
+  tag: "z-tag",
+  styleUrl: "styles.css",
+  shadow: false,
+  scoped: true,
+})
+export class ZTag {
+  /** [optional] Tag icon */
+  @Prop() icon?: string;
+  /** [optional] Hide the text and show it on hover*/
+  @Prop() expandable?: boolean;
+
+  render() {
+    return (
+      <Host
+        class={classNames("body-5-sb", {
+          expandable: this.expandable && this.icon,
+        })}
```

Comment:
stencil sa fare le stesse cose di `classNames` già builtin
```suggestion
        class={{
          "body-5-sb": true,
          expandable: this.expandable && this.icon,
        }}
```